smdebug/tensorflow/keras.py

```diff
@@ -716,6 +716,11 @@ def run(*args, **kwargs):
                 # at this point we need all collections to be ready
                 # this may not be the case at creation of hook
                 # as user's code after hook might add collections
+                self.collection_manager.get(CollectionKeys.WEIGHTS).include(
+                    "^weights/.*/((?!bias).)*$"
+                )
+                self.collection_manager.get(CollectionKeys.LOSSES).include(".*loss.*")
+                self.collection_manager.get(CollectionKeys.GRADIENTS).include("^gradient")
```

Comment:
It doesn't seem like we have tested the regex patterns themselves.

Reply:
copied them from pt regex. do we have tests for regex?